sigmf-spec.md

```diff
@@ -278,8 +278,8 @@ annotation segment objects:
 |`sample_count`|true|uint|The number of samples that this segment applies to. |
 |`generator`|false|string|Human-readable name of the entity that created this annotation.|
 |`comment`|false|string|A human-readable comment.|
-|`freq_lower_edge`|false|double|The lower edge of the frequency band of a signal feature that this annotation describes.|
-|`freq_upper_edge`|false|double|The upper edge of the frequency band of a signal feature that this annotation describes. |
+|`freq_lower_edge`|false|double|The true frequency value (at RF, if applicable) of the lower edge of the feature described by this annotation.|
```

Comment:
I don't disagree; this was my shot at trying to get a discussion going about the best words to use. This one is a bit tricky to make clear while also being succinct.

 @schneider42 @kpreid - How about going with the simplest form possible?
```
The absolute frequency of the lower edge of the feature described by this annotation.
```

Thoughts? If you don't care for it, do you have suggestions for other forms?